__tests__/apiMock/getMockedProperties.test.ts

```diff
@@ -43,66 +43,55 @@ lastName: Smith
   });
 
   it("with array", async () => {
-    const properties = {
-      count: {
-        type: "integer",
-        example: 123,
+    const openApiProperties = {
+      id: {
+        type: "string",
       },
-      results: {
-        type: "array",
-        items: {
-          type: "object",
-          properties: {
-            id: {
-              type: "string",
-              format: "uuid",
-            },
-            level: {
-              enum: [1, 8, 15],
-              type: "integer",
-            },
-          },
-        },
+      name: {
+        type: "integer",
       },
     };
 
-    const expected = {
-      count: 123,
-      results: [
-        {
-          id: "123",
-          level: 1,
-        },
-      ],
-    };
-
     const openAiResponse = `
-        id: John 
-        lastName: Smith
+        id: [1,2,3]
+        name: ['John', 'Eric', 'Martin']
       `;
 
+    const expected = [
+      {
+        id: 1,
+        name: "John",
+      },
+      {
+        id: 2,
+        name: "Eric",
+      },
+      {
+        id: 3,
+        name: "Martin",
+      },
+    ];
+
     (getOpenAIResponse as jest.Mock).mockReturnValue(openAiResponse);
 
     const res = await getMockedProperties(
-      properties,
+      openApiProperties,
       "/api/v1/Status/fullname/123",
       "GET",
-      [
-        {
-          path: ["id"],
-          data: "The user's id",
-        },
-      ]
+      null,
+      undefined,
+      true
     );
-    // Doesnt pass currently
-    // expect(res).toEqual(expected);
+    expect(res).toEqual(expected);
   });
   it("nested object", async () => {
+    // TODO: remove properties from here
+    // learney.atlassian.net/browse/SF-2007
     const openAiResponse = `
   browserSdkVersion: 1
   dateCreated: 2021-01-01
-  cdn: nice-cdn
-  csp: nice-csp
+  dsn.properties.cdn: nice-cdn
+  dsn.properties.csp: nice-csp
```

Comment:
So this still doesn't remove the `properties` and `items` keywords from the generated json? Or am I missing a step where this happens?

Reply:
properties is still there, there's a ticket for this

https://learney.atlassian.net/jira/software/projects/SF/boards/2/backlog?assignee=627e349e98eae500689fcf62&selectedIssue=SF-2007